shell/platform/windows/flutter_window.cc

```diff
@@ -806,12 +806,9 @@ LRESULT FlutterWindow::OnGetObject(UINT const message,
   }
 
   gfx::NativeViewAccessible root_view = GetNativeViewAccessible();
-  // TODO(schectman): UIA is currently disabled by default.
-  // https://github.com/flutter/flutter/issues/114547
```

Comment:
Unfortunately, Flutter's UIA support isn't ready to be enabled. We won't be able to land an approach that enables UIA by default.

Reply:
With UIA disabled, automationId is not exposed. What can be done for UIA enablement ? Could it be optionally enabled when building an application ?

Reply:
Found https://github.com/flutter/engine/pull/37754, which indicates missing parts relate to text edit carat navigation narration. Could UIA be enabled during application build, opposed to current engine build ?